doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,33 @@
+# QoS policies
+
+Unlike other middlewares, e.g. [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html),
+iceoryx keeps things simple and offers basic Quality of Service (QoS) settings. The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings are:
+
+| Option                              | Explanation                                                                                                                                       |
+|-------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PublisherOptions::historyCapacity` | This enables late-joining subscribers to request the n last samples                                                                               |
```

Comment:
This is correct but the implementation is "kaputt".
See #780 

Here we have to also explain the contract in detail. Is the subscriber subscribing to a publisher or a service - I think service is here the right answer but our implementation assumes publisher. We have to also define what a service is.

To summarize the problem:
1. Single publisher - everything works
2. Multi publisher broken - you get the last `n` samples of every publisher so this means for `m` publishers you get currently in the worst case `m * n` samples in random order, not in the order they were published on the service
3. Multi publisher when publisher called stop offer or is removed - If the publisher who send the last n samples is removed or called stop offer you will never get the last `n` samples since they are vanished. You get arbitrary `n` samples at best or nothing.

Reply:
@elfenpiff Added to `iceoptions` readme as this does not belong to the QoS article.